engine/math/vector.lisp

```diff
@@ -0,0 +1,203 @@
+(defpackage :lbge.math)
```

Comment:
If you want use `+` and other operations from `:cl` package, you should add `:use` clause here.